api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/PrepNewCohortQueries.java

```diff
@@ -32,6 +32,25 @@ public PrepNewCohortQueries(
     this.genericCohortQueries = genericCohortQueries;
   }
 
+  /**
+   * O sistema irá produzir o indicador B1 – “Nº de Utentes que iniciaram PrEP pela 1ª vez durante o
+   * período de reporte” da seguinte forma:
+   *
+   * <ul>
+   *   <li>Incluindo todos os utentes que iniciaram PrEP durante o período verificando se a data
+   *       mais antiga entre as seguintes datas ocorre dentro do período (>= “Data Início do
+   *       Relatório” e <= “Data Fim do Relatório”):
+   *       <ul>
+   *         <li>“O(a) utente está a iniciar pela 1a vez a PrEP Data” registada na “Ficha de
+   *             Consulta Inicial PrEP”
+   *         <li>“Data de Início PrEP” registada na secção “Resumo da Ficha de Seguimento” da “Ficha
+   *             de Consulta Inicial PrEP”
+   *       </ul>
+   *   <li>Excluindo todos os utentes “Transferidos De” outra Unidade Sanitária (RF10)
+   * </ul>
+   *
+   * @return CohortDefinition
+   */
```

Comment:
@TinoMuianga please add the correct requirements for this method. Refer to Prep New requirements